docs/antrea-network-policy.md

```diff
@@ -1,4 +1,4 @@
-# Antrea Policy CRDs
+# Antrea Network Policy
```

Comment:
the only reason why i named it Antrea Policy CRDs is to be able to specify that this doc is for both Antrea NetworkPolicy and Antrea ClusterNetworkPolicy.
i think we use  Antrea NetworkPolicy to refer both generic usage and namespaced policies. do you think its not an issue?

Reply:
I understood. But do you think "Antrea Network Policy" can represent both Namespaced NetworkPolicy and ClusterNetworkPolicy?
If you prefer, how about Antrea Network Policy CRDs?

Reply:
sounds good to me

Reply:
Sorry, you like "Antrea Network Policy" or ""Antrea Network Policy CRDs"?

Reply:
"Antrea Network Policy CRDs" :)